templates/vpc.yaml

```diff
@@ -213,58 +225,136 @@ Resources:
           ToPort: 443
           CidrIp: !Ref VpcIpBlock
 
+  ValidateEndpointSubnetFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: validate_vpce_subnets.handler
+      CodeUri: ../backend/lambdas/custom_resources/
+      Description: Custom Lambda resource for the Amazon S3 Find and Forget Cloudformation Stack
+      Policies:
+        - Statement:
+          - Effect: Allow
+            Action:
+              - ec2:DescribeSubnets
+              - ec2:DescribeVpcEndpointServices
+            Resource: '*'
+
   # Endpoints
 
-  CloudWatchEndpoint:
-    Type: AWS::EC2::VPCEndpoint
+  CloudWatchEndpointSubnets:
```

Comment:
Is there an additional endpoint being added here? If so the estimated costs need to be updated

Reply:
No, there are no new endpoints being added. They are the same as before this PR.